backend/app/routers/files.py

```diff
@@ -409,3 +410,43 @@ async def resubmit_file_extractions(
             FileOut(**file.dict()), rabbitmq_client, user, credentials
         )
     return resubmit_success_fail
+
+@router.post("/{file_id}/visualization")
```

Comment:
Since this is a collection resource (i.e., many visualizations for a resource), it may be better to use the plural form (visualizations) in the URL. What do you think?

Reply:
renamed.